Felienne/hedy
# Adds achievement for duplicating a class

**Description**
In this PR we add an achievement "One for the Money, Two for the Show" for duplicating a class. 

**Fixes**
This PR fixes #3124.

**How to test**
Make sure you are a teacher. Duplicate a class and verify that you receive this new achievement.
